ui/src/components/Terminal/index.vue

```diff
@@ -90,7 +90,7 @@ const receivePostMessage = (): void => {
         emits('update:drawer', true, t('Settings'), 'setting');
         break;
       case WINDOW_MESSAGE_TYPE.FILE:
-        emits('update:drawer', true, t('FileManager'), 'file-manager', windowMessage.token.id);
+        emits('update:drawer', true, t('FileManager'), 'file-manager', windowMessage.SFTP_Token);
         break;
       case WINDOW_MESSAGE_TYPE.FOCUS:
         terminal.value?.focus();
```

Comment:
None of the changes in this particular snippet seem to lead to any major issues or irregularities. Therefore, no optimizations suggestions are needed.

However, please note that I will be more helpful if you could clarify the specific area where we need help with these differences: (i) formatting errors like bad indentation or syntax errors, (ii) logical inconsistencies between lines or sections, (iii) missing dependencies or imports, or (iv) potential security vulnerabilities like SQL injection attacks, XSS, etc. If there's anything else unclear, just let me know.